app/hotels/src/reducers/SearchReducer.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+
+import type { ReduxState, SearchActions } from '../types/Redux';
+
+type SearchState = $PropertyType<ReduxState, 'search'>;
+
+const InitialFilter = {
+  isActive: false,
```

Comment:
I am afraid that it's not only about `on/off` but it's more fuzzy. You can for example have `off/"1,2,4"` or `off/>123` and so on...